core/src/main/java/de/danoeh/antennapod/core/storage/FeedCursorMapper.java

```diff
@@ -0,0 +1,70 @@
+package de.danoeh.antennapod.core.storage;
+
+import android.database.Cursor;
+
+import androidx.annotation.NonNull;
+
+import de.danoeh.antennapod.core.feed.Feed;
+import de.danoeh.antennapod.core.feed.FeedPreferences;
+import de.danoeh.antennapod.core.storage.PodDBAdapter;
+import de.danoeh.antennapod.core.util.SortOrder;
+
+/**
+ * Converts a {@link Cursor} to a {@link Feed} object.
+ */
+public class FeedCursorMapper {
+
+    /**
+     * Create a {@link Feed} instance from a database row (cursor).
+     */
+    @NonNull
+    public Feed convert(@NonNull Cursor cursor) {
```

Comment:
Could you please make it `static`? Then we don't need to allocate an additional object every time a cursor is mapped.

Reply:
I got you wrong with using the same interface as CursorMapper. Method is static again.

Reply:
To be honest, I wasn't even thinking about static vs non-static when writing about the interface. Just noticed it later when reviewing the changes :)